integration/setup/config.go

```diff
@@ -18,6 +18,7 @@ const (
 
 type Config struct {
 	HelmClient *helmclient.Client
+	K8sClient  kubernetes.Interface
```

Comment:
K8sClient is used to get the tiller image from the deployment.

Reply:
What do you think about setting up `K8sSetup` below as an internal? We have all dependencies in place and the change is breaking anyway?

Reply:
Yes good point. Done.